packages/common/test/ContractUtils.js

```diff
@@ -1,8 +1,8 @@
 // Script to test
 const ContractUtils = require("../src/ContractUtils");
 const { getTruffleContract, getAbi } = require("@uma/core");
-const ERC20 = getTruffleContract("BasicERC20", web3, "latest");
-const ERC20ABI = getAbi("BasicERC20", "latest");
```

Comment:
Does the `latest` flag cause the tests to unexpectedly use the wrong contract versions?

Reply:
I dont think so as these kinds of contracts have never changed between versions.